### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by swade1987**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @ksoclabs/engineering
```
Comment (benhartley) [inline CODEOWNERS:1]: Showing error here - does team exist?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+initialise:
+	pre-commit install
+	pre-commit run -a
+
+changelog:
+	git-chglog -o CHANGELOG.md --next-tag `semtag final -s minor -o`
+
+release:
+	semtag final -s minor
```
Comment (benhartley) [inline Makefile:9]: I don't think we should fix to a version type. Perhaps we could use `release-%` instead?

File: CODEOWNERS (hunk 1 above)
Comment (swade1987) [reply CODEOWNERS:1]: Yeh its happening on this other repo as well but https://github.com/orgs/ksoclabs/teams/engineering

File: Makefile (hunk 2 above)
Comment (swade1987) [reply Makefile:9]: Makes sense, will align with our other repos.

Comment (swade1987) [reply Makefile]: Fixed.

**Review by benhartley — APPROVED** [review]

**Comment by ksoc-automator** [comment]
This PR is included in version 1.0.0 :tada:

**Pull request merged**